[2024-08-08T06:52:07.208+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-08T06:52:07.254+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_mysql.etl_task scheduled__2024-08-07T00:00:00+00:00 [queued]>
[2024-08-08T06:52:07.265+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_mysql.etl_task scheduled__2024-08-07T00:00:00+00:00 [queued]>
[2024-08-08T06:52:07.265+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-08T06:52:07.300+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): etl_task> on 2024-08-07 00:00:00+00:00
[2024-08-08T06:52:07.333+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=2238) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-08T06:52:07.336+0000] {standard_task_runner.py:64} INFO - Started process 2243 to run task
[2024-08-08T06:52:07.333+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_mysql', 'etl_task', 'scheduled__2024-08-07T00:00:00+00:00', '--job-id', '186', '--raw', '--subdir', 'DAGS_FOLDER/etl_mysql.py', '--cfg-path', '/tmp/tmpfc0w6zgg']
[2024-08-08T06:52:07.337+0000] {standard_task_runner.py:91} INFO - Job 186: Subtask etl_task
[2024-08-08T06:52:07.384+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_mysql.etl_task scheduled__2024-08-07T00:00:00+00:00 [running]> on host bd99345677b7
[2024-08-08T06:52:07.459+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_mysql' AIRFLOW_CTX_TASK_ID='etl_task' AIRFLOW_CTX_EXECUTION_DATE='2024-08-07T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-08-07T00:00:00+00:00'
[2024-08-08T06:52:07.461+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-08T06:52:07.566+0000] {logging_mixin.py:188} INFO - ETL process completed successfully!
[2024-08-08T06:52:07.567+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-08-08T06:52:07.569+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-08T06:52:07.577+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_mysql, task_id=etl_task, run_id=scheduled__2024-08-07T00:00:00+00:00, execution_date=20240807T000000, start_date=20240808T065207, end_date=20240808T065207
[2024-08-08T06:52:07.630+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-08T06:52:07.646+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-08-08T06:52:07.648+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
